### ISSUE TITLE
Video sped up 

### ISSUE THREAD

**Issue by Clicli99** [C0untFloyd/roop-unleashed#194]
Hello, I don’t know If it’s just for me or everyone else , but the final generated video comes sped up and out of sync with the audio. How can I resolve this ?

**Comment by C0untFloyd** [comment]
Very likely this #111

**Comment by ShrekLovesAll** [comment]
Can confirm that I've been having this issue for a few days too. Only specific videos seem to have this problem, others will read the FPS just fine. Rolling ffmpeg back to 5.1.2 didn't help either. When I run a command to probe a video and get back the info, ffmpeg reads it just fine. It just doesn't seem to work in gradio. Hell, I even had a video at 30fps come out as 60fps. 

The only solution I've found so far is to simply reencode the video. The latest 9-04 build of ffmpeg 6.0 didn't help it either. I'm pulling this guess out of my ass but I'd say that there's an issue within Roop and how it handles the output framerate, and I guess there's the issues others mentioned about ffprobe just not giving them the right info. I have absolutely no clue how my 30fps videos came out as 60fps either, since I'm pretty sure 30fps is the fallback value. Keep frames, keep fps, none of those settings seem to have an impact on this result (and I'm not 100% of what exactly they do either). Maybe a manual output fps selector would help? No clue tbh but just reencode the videos as plainly as possible in h.264 and that should hopefully fix it. I had to change one of my videos to 30fps too but that's not exactly ideal before roop since I ended up with some duplicate frames (24fps -> 30fps).
